src/main/msp/msp_protocol.h

```diff
@@ -324,6 +324,8 @@
 #define MSP_SET_GPS_RESCUE_PIDS  226    //in message          GPS Rescues's throttleP and velocity PIDS + yaw P
 #define MSP_SET_VTXTABLE_BAND    227    //in message          set vtxTable band/channel data (one band at a time)
 #define MSP_SET_VTXTABLE_POWERLEVEL 228 //in message          set vtxTable powerLevel data (one powerLevel at a time)
+#define MSP_SET_LED_STRIP_CONFIG_VALUES 231 //in  message     Set LED strip config values
+#define MSP_LED_STRIP_CONFIG_VALUES     232 //out  message    Get LED strip config vaéues
```

Comment:
We already have `MSP_SET_LED_COLORS`.

Reply:
That's for the 16 usable colors, do you want me to extend `MSP_SET_LED_COLORS` message?

Reply:
@klutvott123 What is the right way of doing this? It looks like we soon run out of MSP commands

Reply:
We have MSP2 as extension

Reply:
So what should I do now? it's not clear.

Reply:
If this a global setting then we probably should add MSP2 messages, otherwise extend either MSP_LED_COLORS, LET_STRIP_CONFIG or LED_STRIP_MODECOLOR

Reply:
the led strip messages are already very specific as to what they're for, so I guess a separate message for this is fine. These parameters are part of the ledStripConfig so they COULD go into MSP_LED_STRIP_CONFIG.
We'll run out of MSP messages eventually, but I don't think it's a problem as we already have MSP2. 
I'm fine with either. 🙂

Reply:
@klutvott123 I'm transforming the code to MSP2 now and I can't extend other messages because they are called very frequently as the user pull the slider so I don't want to send every config with 10ms interval